src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -543,7 +543,7 @@ public static string TryGetAndroidJarPath (TaskLoggingHelper log, string platfor
 			var platformPath = MonoAndroidHelper.AndroidSdk.TryGetPlatformDirectoryFromApiLevel (platform, MonoAndroidHelper.SupportedVersions);
 			if (platformPath == null) {
 				var expectedPath = MonoAndroidHelper.AndroidSdk.GetPlatformDirectoryFromId (platform);
-				log.LogError ("XA5207", "Could not find android.jar for API Level {0}. " +
+				log.LogError (null, "XA5207", null, null, 0, 0, 0, 0, "Could not find android.jar for API Level {0}. " +
```

Comment:
Instead of using the *9* parameter `LogError()` method, perhaps we should instead use our `log.LogCodedError()` extension method?